src/app/components/sandbox/Preview/utils/should-update.js

```diff
@@ -24,7 +24,7 @@ function compareArrays(m1, m2, fields: Array<String>) {
       return true;
     }
 
-    return checkFields(first, second, MODULE_FIELDS);
+    return checkFields(first, second, fields);
```

Comment:
Oh this is a good one! So all this time I've been passing in fields that don't get checked?